db/migrations/20200401121544_add_system_user_setting_migration.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Copyright (C) 2020 Xibo Signage Ltd
+ *
+ * Xibo - Digital Signage - http://www.xibo.org.uk
+ *
+ * This file is part of Xibo.
+ *
+ * Xibo is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * Xibo is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Xibo.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+use Phinx\Migration\AbstractMigration;
+
+class AddSystemUserSettingMigration extends AbstractMigration
+{
+    /** @inheritdoc */
+    public function change()
+    {
+        if (!$this->fetchRow('SELECT * FROM `setting` WHERE setting = \'SYSTEM_USER\'')) {
+
+            // get System User
+            $getSystemUserQuery = $this->query('SELECT userId FROM user WHERE userTypeId = 1 ORDER BY userId LIMIT 1');
+            $getSystemUserResult = $getSystemUserQuery->fetchAll(PDO::FETCH_ASSOC);
+
+            $userId = $getSystemUserResult[0]['userId'];
```

Comment:
I know this would be odd (very odd) - but what if there aren't any users?
We probably shouldn't fail to upgrade, even though no-one can login.